src/main/java/com/kodilla/ecommercee/user/controller/UserController.java

```diff
@@ -0,0 +1,37 @@
+package com.kodilla.ecommercee.user.controller;
```

Comment:
Czy nie lepiej jakbyśmy wrzucili wszystkie kontrolery do jednego pakietu "contorller"?

Reply:
To jest kwestia dogadania formy. Dla mnie taka forma jest bardziej przejrzysta, ale może być i tak, jak proponujesz. Może dogadamy wszystko na spotkaniu online.